src/client/components/TetrisView/TetrisView.js

```diff
@@ -10,16 +11,25 @@ import './TetrisView.scss';
 import store from '../../store';
 import { togglePlay } from '../../actions/tetris';
 
-const TetrisView = () => (
-  <div className="container">
-    <div className="tetris-view">
-      <Tetris />
-      <TetrisPlayersList />
+const TetrisView = ({ game }) => {
+
+  console.log('game -- ', game);
+  if (!game) return <Redirect to="/" />;
+
+  return (
+    <div className="container">
+      <div className="tetris-view">
+        <Tetris/>
+        <TetrisPlayersList/>
+      </div>
+      {/* start button for testing only */}
+      <RaisedButton label="start/stop" style={{display: 'inherit'}} onClick={() => store.dispatch(togglePlay())}/>
     </div>
-    {/* start button for testing only */}
-    <RaisedButton label="start/stop" style={{ display: 'inherit' }} onClick={() => store.dispatch(togglePlay())} />
-    <RTCTest />
-  </div>
-);
+  );
+};
+
+const mapStateToProps = state => ({
+  game: state.game,
+});
 
```

Comment:
If we connect this component, shall we disconnect its children and pass them object as props.